### PR TITLE
WIP: Change BMLab provisioning node from server 3 to server 5

### PR DESCRIPTION
**Pull request by Sunnatillo**
Server 3 is facing hardware issues, this will be workaround until the admin fixes the problem

### CODE REVIEW

**Review by fmuyassarov — APPROVED** [review]

File: jenkins/scripts/bare_metal_lab/templates/bmhosts_crs.yaml.j2 [hunk 1]
```diff
@@ -20,14 +20,14 @@ data:
...
+  name: eselda13u31s05.est.lan
 spec:
   online: true
-  bootMACAddress: b4:b5:2f:6d:89:d8
+  bootMACAddress: 80:c1:6e:7a:5a:a8
   bootMode: legacy
   bmc:
-    address: ilo4://192.168.1.12
-    credentialsName: bml-ilo-login-secret-03
+    address: ilo4://192.168.1.14
```
Comment (furkatgofurov7) [inline jenkins/scripts/bare_metal_lab/templates/bmhosts_crs.yaml.j2:29]: this needs  to be changed: https://github.com/metal3-io/project-infra/blob/main/jenkins/scripts/bare_metal_lab/deploy-lab.yaml#L9

**Review by furkatgofurov7 — CHANGES_REQUESTED** [review]

File: jenkins/scripts/bare_metal_lab/templates/bmhosts_crs.yaml.j2 (hunk 1 above)
Comment (Sunnatillo) [reply jenkins/scripts/bare_metal_lab/templates/bmhosts_crs.yaml.j2:29]: Thank you for noticing. Done.

**Comment by furkatgofurov7** [comment]
Using CentOS 8 k8s v1.23.3 node image works fine and tests passed: https://jenkins.nordix.org/job/metal3_project_infra_bml_integration_tests_centos/105/consoleFull
Trying with  CentOS 9 k8s v1.23.3 now

/test-integration-bml-centos

**Comment by furkatgofurov7** [comment]
> Trying with CentOS 9 k8s v1.23.3 now
Failed, trying with CentOS 8 k8s v1.23.5


/test-integration-bml-centos

**Comment by furkatgofurov7** [comment]
> trying with CentOS 8 k8s v1.23.5

Failed

**Comment by furkatgofurov7** [comment]
trying with CentOS 9 k8s v1.24.0
 
 /test-integration-bml-centos

**Comment by furkatgofurov7** [comment]
**Summary of tests:**

- [:heavy_check_mark:] CentOS 8 k8s v1.23.3 node image
- [:x:] CentOS 8 k8s v1.23.5 node image
- [:x:] CentOS 9 k8s v1.23.3 node image
- [:x:] CentOS 9 k8s v1.23.5 node image >>> current main job
- [:x:] CentOS 9 k8s v1.24.0 node image

**Comment by furkatgofurov7** [comment]
@Sunnatillo we do not need this PR anymore AFAIK, can we close it?

**Comment by Sunnatillo** [comment]
> @Sunnatillo we do not need this PR anymore AFAIK, can we close it?

Thanks for reminding. No we do not need this PR anymore. I will close it

**Pull request closed without merging**